### PR TITLE
localCI: implements localCI

### PR DESCRIPTION
**Pull request by devimc**
localCI is a continuous integration system

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
hey

**Comment by devimc** [comment]
Hi

File: cmd/localCI/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- ``url [string]``: the URL to the repository.
+- ``token [string] (optional)``: the repository access token. If the control version repository imposes a rate limit,
+  this token is used to make the requests and post in the pull request the messages specified in ``postOnFailure``
+  and ``postOnSuccess``.
+- ``setup [[]string] (optional)``: the commands needed to setup the test environment.
+- ``run [[]string]``: the commands used to test the pull request once ``setup`` finished correctly.
+- ``teardown [[]string] (optional)``: the commands executed once ``run`` finished correctly.
+- ``runOnSuccess [[]string] (optional)``: the commands to be executed if ``setup``, ``run`` and ``teardown`` finished correctly.
+- ``runOnfailure [[]string] (optional)``: the commands to be executed if any of ``setup``, ``run`` or ``teardown`` fail.
```
Comment (grahamwhaley) [inline cmd/localCI/README.rst]: nit s/runOnfaillure/runOnFailure/ to be consistent

Comment (devimc) [reply cmd/localCI/README.rst]: fixed, thanks :)

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa -passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
qa-passed

**Comment by devimc** [comment]
comment trigger

**Comment by devimc** [comment]
comment trigger does not match

**Comment by devimc** [comment]
comment trigger

**Comment by devimc** [comment]
comment trigger

**Comment by devimc** [comment]
test

**Pull request merged**